js-objects/objects.js

```diff
@@ -0,0 +1,253 @@
+
+/* 
+2.Tаск 2
+
+Створіть обєкт room
+
+Додайте властивості(тут можна одразу записати в обєкт):
+
+name - стрінга, має містити назву кімнати;
+
+size - число, має містити розмір кімнати;
+
+window - булеве значення. // true/false.
+
+Виведіть у консоль значення name, window.
+*/
+
+const room = {
+    name : "living room",
+    size: "1m^2",
+    window: true,
+};
+
+console.log(room["name"]);
+console.log(room["window"]);
+
+/*
+3.Таск 3
+
+Створіть обєкт movie
+
+Додайте властивості(тут можна одразу записати в обєкт):
+
+title - стрінга, має містити назву фільму;
+
+releaseYear - число, має містити дату коли вийшов фільм;
+
+genre - стрінга, має містити жанр.
+
+Замініть жанр фільму, замініть тайтл.
+Додайте 2 додаткові властивості // Наприклад actors: {mainRole : “Test Test”, ….}, director.
+Видаліть title з обєкту movie.
+Виведіть оновлений обєкт movie у консоль.
+*/
+
+const movie = {
+    title: "Titanic",
+    releaseYear: 1997,
+    genre: "drama"
+};
+
+console.log(movie);
+
+movie["genre"] = "action";
+movie["title"] = "Nightcrawler";
+movie["actors"] = { 
+    mainRole: "Jake Gyllenhaal",
+    director: "Dan Gilroy"
+};
+
+delete movie["title"];
+
+console.log(movie);
+
+/*
+4.Таск 4
+
+Дано обєкт:
+
+const englishBooks = [
+    {
+        title: "To Kill a Mockingbird",
+        author: "Harper Lee",
+        year: 1960
+    },
+    {
+        title: "1984",
+        author: "George Orwell",
+        year: 1949
+    },
+    {
+        title: "The Great Gatsby",
+        author: "F. Scott Fitzgerald",
+        year: 1925
+    }
+];
+Пройдіться циклом по масиву обєктів.
+
+Якщо title буде “The Great Gatsby” то пропустіть виконання ітерації.
+Якщо year буде більше ніж 1940, то виведіть у консоль значення ключа та властивості.
+Якщо author “George Orwell” то замініть його на довільне значення.
+Виведіть оновлений масив об’єктів englishBooks
+
+*/
+
+const englishBooks = [
+    {
+        title: "To Kill a Mockingbird",
+        author: "Harper Lee",
+        year: 1960
+    },
+    {
+        title: "1984",
+        author: "George Orwell",
+        year: 1949
+    },
+    {
+        title: "The Great Gatsby",
+        author: "F. Scott Fitzgerald",
+        year: 1925
+    }
+];
+
+// Пройдіться циклом по масиву обєктів.
+
+const iterateObj = (obj) => {
+    for (const key in obj) {
+        console.log(`${key}: ${obj[key]}`);
+    }
+};
+
+for (const book of englishBooks) {
+
+    // Якщо title буде “The Great Gatsby” то пропустіть виконання ітерації.
+    if(book.title === "The Great Gatsby"){
+        continue
+    } 
+
+    // Якщо year буде більше ніж 1940, то виведіть у консоль значення ключа та властивості.
+    if (book.year > 1940) {
+        iterateObj(book);
+    }
+
+    // Якщо author “George Orwell” то замініть його на довільне значення.
+    if (book.author === "George Orwell") {
+        book.author = "Bernard Shaw"; 
+    }
+};
+
+for (const book of englishBooks) {
+    console.log(book)
+};
```

Comment:
You did it more complicated than it is, check pls this suggestion
```suggestion
// // Пройдіться циклом по масиву обєктів.

for (const book of englishBooks) {
    // Якщо title буде “The Great Gatsby” то пропустіть виконання ітерації.
    if (book.title === "The Great Gatsby") {
        continue
    }

    // Якщо year буде більше ніж 1940, то виведіть у консоль значення ключа та властивості.
    if (book.year > 1940) {
        console.log(`Title: ${book.title}, Year: ${book.year}`);
    }

    // Якщо author “George Orwell” то замініть його на довільне значення.
    if (book.author === "George Orwell") {
        book.author = "Bernard Shaw";
    }

}
console.log(englishBooks);
```